Plan/common/src/main/resources/assets/plan/locale/locale_CS.txt

```diff
@@ -435,9 +435,9 @@ HTML - Version Update                           || Aktualizovat
 HTML - Version Update Available                 || Nová verze ${0} je dostupná!
 HTML - Version Update Dev                       || Toto je vývojářská verze.
 HTML - Version Update Info                      || Nová verze je dostupná ke stažení.
-HTML - WARNING_NO_GAME_SERVERS                  || Some data requires Plan to be installed on game servers.
-HTML - WARNING_NO_GEOLOCATIONS                  || Geolocation gathering needs to be enabled in the config (Accept GeoLite2 EULA).
-HTML - WARNING_NO_SPONGE_CHUNKS                 || Chunks unavailable on Sponge
+HTML - WARNING_NO_GAME_SERVERS                  || Některá data vyžadují, aby byla na herní servery nainstalována aplikace Plan.
+HTML - WARNING_NO_GEOLOCATIONS                  || V konfiguraci je třeba povolit shromažďování geolokací (Přijměte GeoLite2 EULA) .
```

Comment:
Is the space at the end of this line a typo?

Reply:
> Is the space at the end of this line (before the dot) a typo?

Yes.

Reply:
You can edit the file here https://github.com/WolverStones/Plan/blob/patch-1/Plan/common/src/main/resources/assets/plan/locale/locale_CS.txt (and commit to `patch-1`) to fix it for this PR